google-pub/c/c.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ldflags": "-lssl -lcrypto -lpaho-mqtt3cs -ljwt -o mqtt_ciotc",
```

Comment:
We need to remove -o mitt_ciotc (it's an option for getting the executable).

Reply:
Done.